api/modelgeneration/modelgeneration.go

```diff
@@ -27,9 +26,9 @@ func NewClient(st base.APICallCloser) *Client {
 }
 
 // AddGeneration adds a model generation to the config.
-func (c *Client) AddGeneration(model names.ModelTag) error {
+func (c *Client) AddGeneration(modelUUID string) error {
```

Comment:
I poked the tag conversion in here for the following reasons:
- It keeps the commands cleaner, as they don't need to do conversion for what is essentially a wire format.
- We only have to import `names` in one file and do all conversion immediately pre-flight.